### DISCUSSION TITLE
Merge latest PR into intel/fpga-ofs-dev-5.15-lts branch

### DISCUSSION THREAD

**Discussion by Muralix** [OFS/linux-dfl-backport#65] [General]
@rchriste1 - you have any comment from your testing on the recent PR #64 ? Shall i merge into intel/fpga-ofs-dev-5.15-lts branch?

**Comment by rchriste1** [comment]
I have tested the merge of https://github.com/OFS/linux-dfl-backport/pull/64 into the intel-test/fpga-ofs-dev-5.15-lts branch. It did not show any issues. So it should be fine to merge into intel/fpga-ofs-dev-5.15-lts branch.

**Comment by rweight** [comment]
@rchriste1 : I just pushed additional changes to intel-test/fpga-ofs-dev-5.15-lts. New commit ID is d63b7fda6877cf5203ae5ca8c1c51b0230d5db8e. These changes have been tested by Intel. Please confirm that these changes do not introduce a regression for you and then we will mirror the changes to intel/fpga-ofs-dev-5.15-lts.
